### PR TITLE
add dApp staking top page background

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

- Added the background image for the leaderboard and ads section
   (I'll come back for background animation later)
- Added each category head image
- Made the dApps slider 2 rows

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Changes**

Screenshot 2024-01-15 at 3 10 33 PM

Screenshot 2024-01-15 at 3 10 44 PM

### CODE REVIEW

File: src/staking-v3/components/Dapps.vue [hunk 1]
```diff
@@ -88,7 +105,15 @@ export default defineComponent({
       return result;
     });
 
-    return { filteredDapps, getDappTier, getDappPageUrl };
+    const bg_img = {
```
Comment (bobo-k2) [inline src/staking-v3/components/Dapps.vue:108]: use camel case and more descriptive variable names. eg, `categoryBackgroundImages`

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/staking-v3/components/Dapps.vue (hunk 1 above)
Comment (ayumitk) [reply src/staking-v3/components/Dapps.vue]: @bobo-k2 fixed in [3e84327](https://github.com/AstarNetwork/astar-apps/pull/1132/commits/3e84327ff48fc254e74abb2b6277083f40a8386e)

**Comment by Kahonnohak** [comment]
@ayumitk could you add a little space here at the bottom? Desk top look okay but mobile look little too tight. maybe 40px should help. it can be added to all sizes.
Screenshot 2024-01-18 at 09 49 41

**Comment by Kahonnohak** [comment]
Would you add the grid too?

**Review by bobo-k2 — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]

**Pull request merged**